routers/api/v1/api.go

```diff
@@ -573,6 +573,10 @@ func RegisterRoutes(m *macaron.Macaron) {
 					m.Get("/status", repo.GetCombinedCommitStatusByRef)
 					m.Get("/statuses", repo.GetCommitStatusesByRef)
 				})
+				m.Group("/git", func() {
```

Comment:
permission check?

Reply:
what exact permission check are you talking about. (It is checked for access read only rights for repo). Do you mean unit type check?

Reply:
Didn't find the access read check. Unit type check should be fixed by my PR #5314 .

Reply:
It is already checked in upper group in function `repoAssignment`

Reply:
Right. It's different from the middleware which has the same name and used by ui routers.